### ISSUE TITLE
cgroup-bin removed in Ubuntu 19.10

### ISSUE THREAD

**Issue by jamestoyer** [ansible-community/ansible-nomad#75]
When running the role on Ubuntu 19.10, I get the following error with ensuring the required OS packages are installed
```
failed: [lab] (item=cgroup-bin) => {"ansible_loop_var": "item", "changed": false, "item": "cgroup-bin", "msg": "No package matching 'cgroup-bin' is available"}
```

After a bit of digging I found that it appears the problem is caused by `cgroup-bin` being deprecated and subsequently removed in 19.10 (see [here](https://unix.stackexchange.com/questions/528516/why-does-deborphan-list-cgroup-bin)).

Looking around it seems that the packages are hard-coded in `vars/Debian.yaml`:
```yaml
nomad_os_packages:
  - cgroup-bin
  - curl
  - git
  - libcgroup1
  - unzip
```

I've tried many ways of [overriding these variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable) but it looks like I'm stuck using `--extra-vars` from the command line or importing and changing the role locally to replace `cgroup-bin` with `cgroup-tools`. While the former is ok, the later feels very wrong. I would put in a PR to update them, but I'm not sure what the best approach to take is as I'm a relative newcomer to Ansible.

**Comment by lanefu** [comment]
yes may need a value based on `ansible_distribution_release`

**Comment by lanefu** [comment]
resolved by #108
